### PR TITLE
zh-CN:tutorials/quickstart/advanced.ipynb

### PR DESCRIPTION
**Pull request by Berrywrq**

### CODE REVIEW

File: site/zh-cn/tutorials/quickstart/advanced.ipynb [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "DUNzJc4jTj6G"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/beta/tutorials/quickstart/advanced\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />在 TensorFlow.org 上查看</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/advanced.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/beta/tutorials/quickstart/advanced\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorflow.google.cn 上查看</a>\n",
```

File: site/zh-cn/tutorials/quickstart/advanced.ipynb [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/advanced.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
```

File: site/zh-cn/tutorials/quickstart/advanced.ipynb [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/advanced.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
```

File: site/zh-cn/tutorials/quickstart/advanced.ipynb [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/r2/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />下载 notebook</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/quickstart/advanced.ipynb]:
```suggestion
    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/beta/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />下载 notebook</a>\n",
```

**Comment by Mr-Linus** [comment]
Sorry to bother you, I want to ask if the disclaimer you refer to is copyright or Licenses or Both.

File: site/zh-cn/tutorials/quickstart/advanced.ipynb (hunk 4 above)
Comment (Berrywrq) [reply site/zh-cn/tutorials/quickstart/advanced.ipynb]: Thanks for reviewing，I‘ve made some modifications to your opinion， please check.

File: site/zh-cn/tutorials/quickstart/advanced.ipynb (hunk 2 above)
Comment (Berrywrq) [reply site/zh-cn/tutorials/quickstart/advanced.ipynb]: I guess “https://www.tensorflow.org/images/colab_logo_32px.png”  should be modified to “https://tensorflow.google.cn/images/colab_logo_32px.png”,  the following two urls are also like this.  Please let me know if I am wrong.

**Comment by lamberta** [comment]
@Mr-Linus Hi. Mentioned in other PR, but the community translation disclaimer should be added at the top of the translated docs. Thanks for all your reviews and help!

**Pull request merged**